mullvad-daemon/src/lib.rs

```diff
@@ -2660,6 +2692,45 @@ impl TunnelParametersGenerator for MullvadTunnelParametersGenerator {
     }
 }
 
+/// Contains all relays that were selected last time when tunnel parameters were generated.
+enum LastSelectedRelays {
+    /// Represents all relays generated for a WireGuard tunnel.
+    /// The traffic flow can look like this:
+    ///     client -> obfuscator -> entry -> exit -> internet
+    /// But for most users, it will look like this:
+    ///     client -> entry -> internet
```

Comment:
Since the `wg_entry` is optional, but `wg_exit` is not. I'd argue that this should be `client -> exit -> internet` if anything.

If we are going to use this representation I really think it should explain that `obfuscator` both can be and not be a bridge. Be and not be an extra hop.